Contents/Code/siteRealityLovers.py

```diff
@@ -0,0 +1,121 @@
+import PAsearchSites
+import json
+import re
+
+def search(results, encodedTitle, title, searchTitle, siteNum, lang, searchByDateActor, searchDate, searchSiteID):
+    Log('******SEARCH CALLED*******')
+    if searchSiteID != 9999: siteNum = searchSiteID
+
+    url = PAsearchSites.getSearchSearchURL(siteNum)
+    post_values = {
+        "sortBy": "MOST_RELEVANT",
+        "searchQuery": searchTitle,
+        "categoryId": "null",
+        "perspective": "null",
+        "actorId": "null",
+        "offset": "0",
+        "isInitialLoad": "true",
+        "videoView": "MEDIUM",
+        "device": "DESKTOP"
+    }
+    params = json.dumps(post_values)
+    req = urllib.Request(url, data=params, headers={'content-type': 'application/json;charset=UTF-8'})
+    data = urllib.urlopen(req).read()
+    searchResults = json.loads(data)
+
+    for searchResult in searchResults['contents']:
+        releaseDate = parse(searchResult['released']).strftime('%b %d %Y')
+        curID = searchResult['videoUri'].replace('/', '_').replace('?', '!')
+        score = 100 - Util.LevenshteinDistance(searchTitle.lower(), title.lower())
+        siteName = PAsearchSites.getSearchSiteName(siteNum)
+        titleNoFormatting = '%s [%s] %s' % (searchResult['title'], siteName, releaseDate)
+        results.Append(MetadataSearchResult(id='%s|%d' % (curID, siteNum), name=titleNoFormatting, score=score, lang=lang))
+
+    return results
+
+
+def update(metadata, siteID, movieGenres, movieActors):
+    Log('******UPDATE CALLED*******')
+
+    id = str(metadata.id).split('|')
+    videoUri = id[0].replace('_', '/')
+    url = PAsearchSites.getSearchBaseURL(siteID) + videoUri
+    detailsPageElements = HTML.ElementFromURL(url)
+
+    metadata.collections.clear()
+    movieGenres.clearGenres()
+    movieActors.clearActors()
+
+    # Studio
+    metadata.studio = PAsearchSites.getSearchSiteName(siteID)
+
+    # Title
+    metadata.title = detailsPageElements.xpath('//h1[@class="video-detail-name"]')[0].text_content().strip()
+
+    # Summary
+    rawSummary = detailsPageElements.xpath('//p[@itemprop="description"]')[0].text_content()
+    formattedSummary = re.sub(' +', ' ', rawSummary.replace("…", "").replace("Read more", "").replace("\n", ""))
+    metadata.summary = formattedSummary
+
+    # Tagline and Collection(s)
+    metadata.tagline = metadata.studio
+    metadata.collections.add(metadata.studio)
+
+    # Genres
+    genres = detailsPageElements.xpath('//span[@itemprop="keywords"]/a')
+    if len(genres) > 0:
+        for genreLink in genres:
+            genreName = genreLink.text_content().strip().lower()
+            movieGenres.addGenre(genreName)
+
+    # Release Date
+    date = detailsPageElements.xpath('//span[@class="videoClip__Details-infoValue"]')[0].text_content().strip()
+    if len(date) > 0:
+        unformattedDate = parse(date).strftime('%b %d %Y')
+        date_object = datetime.strptime(unformattedDate, '%b %d %Y')
```

Comment:
Just `date_object = parse(date)`

Reply:
I think that didn't work because the date is sent with am ordinal format `Jul 01st 2017`